src/targets/gpu/lowering.cpp

```diff
@@ -180,7 +183,7 @@ struct miopen_apply
         add_extend_op("softmax");
         add_extend_op("topk");
 
-        add_gemm_op<op::dot>("dot");
+        add_gemm_dot_op("dot");
```

Comment:
What is the reason to rename this function to "add_gemm_dot_op"? Gemm and dot is the same thing.

Reply:
`quant_dot` and `dot` share the same `add_gemm_op`. But logic was difficult to separate. So, for now, i had to create a duplicate method `add_gemm_dot_op`. I tried template specialization but ran into tidy error saying that method parameter `name` should be made const reference.

Reply:
Done